infra/modules/notifications-email-domain/resources/dns.tf

```diff
@@ -24,7 +22,16 @@ resource "aws_route53_record" "mx_receive" {
   allow_overwrite = true
   type            = "MX"
   ttl             = "600"
+  zone_id         = var.hosted_zone_id
   name            = local.mail_from_domain
+  records         = ["10 feedback-smtp.${data.aws_region.current.name}.amazonses.com"]
+}
+
+resource "aws_route53_record" "dmarc" {
+  allow_overwrite = true
+  ttl             = "600"
+  type            = "TXT"
   zone_id         = var.hosted_zone_id
-  records         = ["10 feedback-smtp.${data.aws_region.current.name}.amazonaws.com"]
+  name            = aws_sesv2_email_identity_mail_from_attributes.sender_domain.mail_from_domain
+  records         = ["v=DMARC1; p=none;"]
 }
```

Comment:
Non-blocking style suggestion. For email I think dns.tf doesn't really say what it's doing, more how it's doing it. What do you think about instead separating into 3 files:

- mail_from.tf
- dkim.tf
- dmarc.tf

mail_from.tf will have the resources:
- local mail_from variable
- aws_sesv2_email_identity_mail_from_attributes.sender_domain
- aws_route53_record.mx_receive
- aws_route53_record.spf_mail_from

dkim.tf would have:
- aws_route53_record.dkim

and dmarc.tf would have:
- aws_route53_record.dmarc

thoughts?

Reply:
While that's probably correct from the point of view of the greater organization of the project, I think it's important to keep the UX in mind here. Splitting up the DNS into 3 files will probably confuse people more often than not. Especially if they don't have strong working knowledge of have email verification works (I certainly don't!)

Reply:
That's reasonable